gems/pending/appliance_console/logfile_configuration.rb

```diff
@@ -8,38 +8,91 @@ module ApplianceConsole
   class LogfileConfiguration
     LOGFILE_DIRECTORY = Pathname.new("/var/www/miq/vmdb/log").freeze
     LOGFILE_NAME = "miq_logs".freeze
+    MIQ_LOGS_CONF = Pathname.new("/etc/logrotate.d/miq_logs.conf").freeze
 
-    attr_accessor :disk
+    attr_accessor :size, :disk, :logrotate_count,
+                  :confirmed_new_disk, :requested_new_disk, :activated_new_disk,
+                  :confirmed_new_logrotate_count, :requested_new_logrotate_count, :activated_new_logrotate_count
 
     include ApplianceConsole::Logging
 
+    def initialize
+      self.requested_new_disk            = false
+      self.confirmed_new_disk            = false
+      self.activated_new_disk            = false
+      self.requested_new_logrotate_count = false
+      self.confirmed_new_logrotate_count = false
+      self.activated_new_logrotate_count = false
+
+      params = { "-h" => nil, "--output=size" => nil, nil => LOGFILE_DIRECTORY }
+
+      self.size = AwesomeSpawn.run!("df", :params => params).output.split("\n")[1].strip
+      self.logrotate_count = File.readlines(MIQ_LOGS_CONF).detect { |line| line =~ /rotate\s*\d.*$/ }.split(" ")[1].to_i
+    end
+
     def activate
-      stop_evm
-      initialize_logfile_disk
-      start_evm
+      activated_new_disk = activate_new_disk if confirmed_new_disk
+      activated_new_logrotate_count = activate_new_logrotate_count if confirmed_new_logrotate_count
+      activated_new_disk || activated_new_logrotate_count
     end
 
     def ask_questions
       clear_screen
-      return false unless use_new_disk
-      choose_disk
+      choose_disk if use_new_disk
+      choose_logrotate_count if set_new_logrotate_count
       confirm_selection
     end
 
     private
 
     def confirm_selection
-      agree("Continue with disk: #{disk.path}: #{disk.size.to_i / 1.megabyte} MB (Y/N):")
+      if requested_new_disk
+        self.confirmed_new_disk =
+          agree("Confirm continue with new logfile disk: #{disk.path}: #{disk.size.to_i / 1.megabyte} MB (Y/N):")
+      end
+
+      if requested_new_logrotate_count
+        self.confirmed_new_logrotate_count =
+          agree("Confirm update the save logrotate count to: #{logrotate_count} (Y/N):")
+      end
```

Comment:
Can we make this one prompt that prints all of the information the user has entered and asks them to confirm?

This is odd that they would need to confirm twice if they wanted to do both things.
Also it looks like we could get rid of the separate `confirmed_*` variables and then just configure what we have information for later.